pkgs/build-support/rust/rust-utils.nix

```diff
@@ -0,0 +1,332 @@
+# Copyright 2017 Pierre-Étienne Meunier
```

Comment:
We have https://github.com/NixOS/nixpkgs/blob/master/COPYING

Reply:
This would be then MIT, right?

Reply:
I removed the copyright notice, it's indeed MIT. Is there a contributors list somewhere?

Reply:
There is a maintainers list in `lib/maintainers.nix`, when you add yourself to a package as maintainer in `meta.maintainers`.